string gamma_subject
string gamma_object
string nameRelation
float64 degreeRelation
